executorlib/standalone/serialize.py

```
@@ -30,7 +30,7 @@ def cloudpickle_register(ind: int = 2):
 
 def serialize_funct_h5(
     fn: Callable, fn_args: list = [], fn_kwargs: dict = {}, resource_dict: dict = {}
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)